test/info.py

```diff
@@ -78,7 +78,11 @@
     ("sam3u2c_ublox_evk_nina_b1_if",                True,       0x5000,     "bin"       ),
     ("lpc11u35_vbluno51_if",                        False,      0x0000,     "bin"       ),
     ('lpc11u35_mtconnect04s_if',                    False,      0x0000,     "bin"       ),
+<<<<<<< HEAD
     ('lpc11u35_mbed_cloud_connect_if',              False,      0x0000,     "bin"       ),
+=======
+    ('tmpm366_tmpm066_stk_if',                      None,       0x10000,    "bin"       ),
+>>>>>>> Add Support for Toshiba TMPM066/TMPM366
```

Comment:
Looks like you have a merge conflict here. That may be why you can't run testing.